helm/fdb-operator/templates/manager/statefulset.yaml

```diff
@@ -63,4 +63,4 @@ spec:
         - containerPort: 9876
           name: webhook-server
           protocol: TCP
-      terminationGracePeriodSeconds: 10
\ No newline at end of file
+      terminationGracePeriodSeconds: 10
```

Comment:
Not needed.

Reply:
I think it is needed but not related to this PR